test/apps/mmap/mmap_and_mremap.c

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MPL-2.0
+
```

Comment:
This test does not appear to be currently included in the CI. You should also add this test to `test/apps/scripts/process.sh` to enable it to run in the CI.